_posts/2025-08-19-cockpit-345.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 345
+author: mvo
+date: '2025-08-19'
+tags: cockpit, machines
+slug: cockpit-345
+category: release
+summary: 'IPv6 for Wireguard, detachable VNC console'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 345 and cockpit-machines 338:
+
+
+## Networking: Support IPv6 addresses for Wireguard
+
+The Networking page can now configure IPv6 addresses for WireGuard VPN
+directly within the WireGuard modal.
+
+<img width="840" height="684" alt="Screen Shot 2025-08-20 at 14 16 46" src="https://github.com/user-attachments/assets/9453ee37-12b8-4882-b8dd-b638cdc1b606" />
+
+## Machines: The VNC console of a virtual machine can be detached
+
+If expanding it is not enough, you can now also detach the VNC console
+of a virtual machine. This allows it to use almost all pixels of a
+browser window (or tab). The window can then of course be made
```

Comment:
This feels a bit too casual... and a bit long-winded...

Reply:
I have added a picture as well.